packages/block-library/src/spacer/editor.scss

```diff
@@ -1,4 +1,4 @@
-.block-library-spacer__resize-container.is-selected {
```

Comment:
Is there any risk a theme may be targeting the is-selected style to change the look of the spacer in the editor?

Reply:
I can bring it back for backward compatibility. It’s hard to tell if it’s necessary 😃